### DISCUSSION TITLE
嗯呢嗯呢

### DISCUSSION THREAD

**Discussion by Difkcjxj** [share121/inter-knot#3290] [其他]
嗯嗯呢嗯呢

**Comment by shan-dgha** [comment]
嗯嗯<img src="https://img-static.mihoyo.com/communityweb/upload/1a25d24ab81aaa88543ffad7cbf3f944.png" alt="绝区零_暗中观察" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/1a25d24ab81aaa88543ffad7cbf3f944.png" alt="绝区零_暗中观察" width="75" >
